assets/src/application/ui/input/InlineEditInput/InlineEditTextArea.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import Editable from './Editable';
+import { TextAreaProps } from './types';
+
+const InlineEditTextArea: React.FC<TextAreaProps> = ({ ...rest }) => {
+	return <Editable inputType='textarea' ellipsis={{ rows: 3, expandable: true }} {...rest} />;
```

Comment:
same here, let's pass `rest` first, this way we will be sure that input type isn't overriden
```
	return <Editable {...rest} inputType='textarea' ellipsis={{ rows: 3, expandable: true }} />;
```

Reply:
Consumers should be able to make any customizations that may be necessary.